.github/workflows/test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        ruby: ["2.6", "2.7", "3.0"]
+        ruby: ["2.7", "3.0", "3.1", "3.2"]
```

Comment:
Ruby 2.6 tests were failing due to an incompatibility with ActiveSupport 6.1. Plus, its [EOL was in 2022](https://www.ruby-lang.org/en/downloads/branches/) so I figured we're safe to stop supporting it.